src/OrbitQt/include/OrbitQt/MoveFilesProcess.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_QT_MOVE_FILES_PROCESS_H_
+#define ORBIT_QT_MOVE_FILES_PROCESS_H_
+
+#include <QObject>
+#include <QThread>
+#include <filesystem>
+
+namespace orbit_qt {
+
+class MoveFilesProcess : public QObject {
+  Q_OBJECT
+
+ public:
+  explicit MoveFilesProcess();
+  ~MoveFilesProcess() noexcept override;
+
+  void Run();
```

Comment:
Nit: To me `Run` sounds like a synchronous function. I would prefer either `Start` or something like `RunAsync`.